src/Entity/SmsMessage.php

```diff
@@ -136,6 +144,21 @@ public function removeOption($name) {
   /**
    * {@inheritdoc}
    */
+  public function getResult() {
```

Comment:
Just a consideration: is it worth it to make a trait out of code shared between Message/SmsMessage and Entity/SmsMessage